x-pack/plugins/security_solution/common/endpoint/index_data.ts

```diff
@@ -32,12 +32,11 @@ import {
 } from '../../../ingest_manager/common';
 import { factory as policyConfigFactory } from './models/policy_config';
 import { HostMetadata } from './types';
+import { KbnClientWithApiKeySupport } from '../../scripts/endpoint/resolver_generator_script';
```

Comment:
This import from `resolver_generator_script.ts` is causing a cyclic dependency since that file also imports from here: https://github.com/elastic/kibana/pull/74305/commits/56d2c996c3b27d40e263dbe7841c0fd504583f76#diff-52d0282fedafcd94a4ef5a5021c90c80R14